recommender/engine.go

```diff
@@ -98,6 +100,14 @@ type NodePool struct {
 	VmClass string `json:"vmClass"`
 }
 
+// ClusterRecommendationAccuracy encapsulates recommendation accuracy
+type ClusterRecommendationAccuracy struct {
+	RespMem  float64  `json:"memory"`
```

Comment:
rename these fields to better signal what they are (also document them: eg.:  RecMem (recommended memory) etc ...